build-tools-internal/src/main/groovy/elasticsearch.run.gradle

```diff
@@ -29,7 +29,8 @@ testClusters.register("runTask") {
       systemProperty 'ingest.geoip.downloader.enabled.default', 'true'
       setting 'xpack.security.enabled', 'true'
       keystore 'bootstrap.password', 'password'
-      user username: 'elastic-admin', password: 'elastic-password', role: 'superuser'
+      user username: 'elastic-admin', password: 'elastic-password', role: '_root'
+      extraConfigFile 'roles.yml', file('src/main/resources/roles.yml')
```

Comment:
This does not work for me when running `./gradlew :run`. The error is file not found. Looks like it tries to find the file under the root project:
> A problem was found with the configuration of task ':run' (type 'RunTask').
>  In plugin 'org.elasticsearch.gradle.DistributionDownloadPlugin' type 'org.elasticsearch.gradle.testclusters.RunTask' property 'clusters.runTask$0.nodes.$0.extraConfigFiles.roles.yml$0.file' specifies file '/Users/ywangd/projects/elastic/elasticsearch/src/main/resources/roles.yml' which doesn't exist.

It works if I change it to
```
extraConfigFile 'roles.yml', file('build-tools-internal/src/main/resources/roles.yml')
```

But this file does not seem to be necessary anyway because the role `_es_test_root` is always added if any user is provisioned. This means we can just have:
```
user username: 'elastic-admin', password: 'elastic-password', role: '_es_test_root'
```

I think it is also preferable to have this special role defined once in a single place.

Reply:
Thank for checking.
This was an early failure & fix from before the new standard role was added to `testclusters`. 

Now that we have a standardised way to solve this, I'll switch it across.